cli/pub_command.go

```diff
@@ -130,13 +135,13 @@ func (c *pubCmd) prepareMsg(body []byte, seq int) (*nats.Msg, error) {
 }
 
 func (c *pubCmd) doReq(nc *nats.Conn, progress *uiprogress.Bar) error {
+
 	for i := 1; i <= c.cnt; i++ {
-		start := time.Now()
 		if !c.raw && progress == nil {
 			log.Printf("Sending request on %q\n", c.subject)
 		}
 
-		body, err := pubReplyBodyTemplate(c.body, i)
+		body, err := pubReplyBodyTemplate(c.body, 0)
```

Comment:
I think the `0` here should be `i`, what was the reason for the change? 

This is the value that `{{Count}}` would set in the body template and its the number of messages sent, setting this to 0 always will break that feature.

Reply:
That was a bug from an earlier iteration - thanks for catching that - will fix.